### PR TITLE
Google Memcached hooks - improve protobuf messages handling

### PR DESCRIPTION
**Pull request by TobKed**
Use to_dict method introduced in `proto-plus 1.11.0` instead if operating on json

Related links:
https://github.com/googleapis/python-memcache/issues/19#issuecomment-713664513
https://github.com/googleapis/proto-plus-python/pull/154
https://proto-plus-python.readthedocs.io/en/latest/messages.html#serialization

### CODE REVIEW

**Comment by TobKed** [comment]
PTAL @mik-laj @tanjinP

File: airflow/providers/google/cloud/hooks/cloud_memorystore.py [hunk 1]
```diff
@@ -572,7 +571,7 @@ def _append_label(instance: cloud_memcache.Instance, key: str, val: str) -> clou
     @staticmethod
     def proto_message_to_dict(message: proto.Message) -> dict:
         """Helper method to parse protobuf message to dictionary."""
-        return json.loads(message.__class__.to_json(message))
+        return message.__class__.to_dict(message)
```
Comment (turbaszek) [inline airflow/providers/google/cloud/hooks/cloud_memorystore.py]: Can you add comments in the code explaining why?

**Review by tanjinP — APPROVED** [review]

File: airflow/providers/google/cloud/hooks/cloud_memorystore.py (hunk 1 above)
Comment (mik-laj) [inline airflow/providers/google/cloud/hooks/cloud_memorystore.py]: Do we need this method at all? I think we can replace these methods with direct calls.

Comment (TobKed) [reply airflow/providers/google/cloud/hooks/cloud_memorystore.py]: I was thinking about it as well. Please check last updates to the PR.

**Review by mik-laj — APPROVED** [review]

File: airflow/providers/google/cloud/hooks/cloud_memorystore.py (hunk 1 above)
Comment (mik-laj) [reply airflow/providers/google/cloud/hooks/cloud_memorystore.py]: This code has been deleted. Can you look again?

**Review by turbaszek — APPROVED** [review]

**Pull request merged**